broker-daemon/state-machines/plugins/state-machine-events.js

```diff
@@ -0,0 +1,69 @@
+const EventEmitter = require('events')
+
+const StateMachinePlugin = require('./abstract')
+
+/**
+ * Events that should trigger the cleanup of all event handlers created
+ * by the StateMachineEvents plugin
+ *
+ * @constant
+ * @type {Array<String>}
+ * @default
+ */
+const EXIT_EVENTS = Object.freeze(['cancel', 'complete', 'reject'])
```

Comment:
I think these plugins should be generic, something we would be comfortable using for any state machine.

This should be pulled into configuration for the plugin.